common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4508,7 +4508,7 @@ public static enum ConfVars {
         "If enabled, HiveServer2 will block any requests made to it over http " +
             "if an X-CSRF-TOKEN header is not present"),
     HIVE_SECURITY_COMMAND_WHITELIST("hive.security.command.whitelist",
-      "set,reset,dfs,add,list,delete,reload,compile,llap",
+      "set,reset,dfs,add,list,delete,reload,compile,llap,processlist",
```

Comment:
IMO, this command should only be executed by admin users. Other users should be restricted from running this command.

Reply:
This command can be run by any user. We have removed query string from display and it does not contain any other sensitive info.

Reply:
can we return extra details like query string for admin users?

Reply:
Yes, I have left a TODO comment for authorization. We can authorize the admin user and display query String or any other sensitive info in follow-up PR.